v03_pipeline/lib/annotations/snv_indel.py

```diff
@@ -80,9 +80,9 @@ def gnomad_non_coding_constraint(
         z_score=(
             interval_ht.index(ht.locus, all_matches=True)
             .filter(
-                lambda x: hl.is_defined(x.gnomad_non_coding_constraint['z_score']),
+                lambda x: hl.is_defined(x['z_score']),
```

Comment:
These no longer take in the `interval_ht` right?  So the dependency itself change 🤔

Reply:
The `interval_ht` is the `screen ht` and `ht` is the annotations table. I could rename the `interval_ht` arg to be representative of the dataset

Reply:
gotcha, yeah.  I think making the arg name representative of the dataset is worthwhile now that the ht itself is dataset specific!